engine/lib/phx/script/ffi_gen/Engine.lua

```diff
@@ -33,6 +34,7 @@ function Loader.defineType()
         Engine = {
             Window     = libphx.Engine_Window,
             Input      = libphx.Engine_Input,
+            Hmgui      = libphx.Engine_Hmgui,
```

Comment:
no PascalCase? :)

Reply:
Forgot that we can rename methods for FFI. Fixed.